packages/conveyer/src/Conveyer.ts

```diff
@@ -312,10 +316,22 @@ class Conveyer extends Component<ConveyerEvents> {
   public updateItems() {
     const scrollAreaElement = this._scrollAreaElement;
     const itemSelector = this._options.itemSelector;
+    const childrenDiffer = this._childrenDiffer;
```

Comment:
It's good to use children differ. Rather than using classes/instances. Use the function `import { diff } from "@egjs/children-differ".